VOKMockUrlProtocol.m

```diff
@@ -49,6 +49,19 @@ + (instancetype)containerWithResponse:(NSHTTPURLResponse *)response data:(NSData
 
 @implementation VOKMockUrlProtocol
 
+- (instancetype)initWithRequest:(NSURLRequest *)request
+                 cachedResponse:(NSCachedURLResponse *)cachedResponse
+                         client:(id<NSURLProtocolClient>)client
+{
+    if (self = [super initWithRequest:request
+                       cachedResponse:cachedResponse
+                               client:client]) {
+        self.mockDataDirectory = DefaultMockDataDirectory;
```

Comment:
I think, ideally, rather than setting this to the default, either override the getter to return the default when the property itself is `nil` or at the point(s) where it's used, use the default as a fall-back when the property is `nil`.  That way, setting the property back to `nil` causes it to go back to its default value (like [`null_resettable`](http://nshipster.com/swift-1.2/#nullability-annotations)).